mffpy/tests/test_writer.py

```diff
@@ -110,3 +111,58 @@ def test_writer_exports_JSON():
         remove(filename)
     except BaseException:
         raise AssertionError(f"""Clean-up failed of '{filename}'.""")
+
+##START##
+
+def test_streaming_writer_receives_bad_init_data():
+    """Test bin writer fails when initialized with non-int sampling rate"""
+    dirname = 'testdir.mff'
+    makedirs(dirname)
+    StreamingBinWriter(100, mffdir=dirname)
+    with pytest.raises(AssertionError):
+        StreamingBinWriter(100.0, mffdir=dirname)
+    rmtree(dirname)
+
+def test_streaming_writer_writes():
+    dirname = 'testdir2.mff'
+    # create some data and add it to a binary writer
+    device = 'HydroCel GSN 256 1.0'
+    num_samples = 10
+    num_channels = 256
+    sampling_rate = 128
+    # create an mffpy.Writer and add a file info, and the binary file
+    W = Writer(dirname)
```

Comment:
The meaning is obvious within this context, but let's avoid single letter variable names as a matter of practice (as annoying and self-referential that kind of is for such aptly named classes).  Maybe
`test_writer` for `W`, `str_bin_writer` for `b`, `test_reader` for `R`.

Reply:
Sorry, another case of just copying the testcase and only changing what needed to be changed.